### PR TITLE
Makefile: allow filtering the system tests by using an environment variable

### PR DESCRIPTION
**Pull request by ygalblum**
#### Does this PR introduce a user-facing change?
No
```release-note
None
```

### CODE REVIEW

**Comment by rhatdan** [comment]
I wait for @edsantiago to chime in on this one.

File: test/README.md [hunk 1]
```diff
@@ -146,6 +146,15 @@ When `bats` is installed and is in your `PATH`, you can run the test suite with
...
 
+## Running a subset of the system tests
+You can filter the tests that will be excuted as part of the suite by setting the environment variable `LOCAL_SYSTEM_FILTER`.
+For example, in order to run only quadlet tests execute the following command:
+```
+LOCAL_SYSTEM_FILTER="quadlet" make localsystem
+```
+
+Please see the man page for `bats` for the description of the `--filter` argument
+
```
Comment (TomSweeneyRedHat) [inline test/README.md:157]: It would be good to add a note about the REMOTE_SYSTEM_FILTER too.

**Comment by edsantiago** [comment]
What does this do that isn't handled better by https://github.com/containers/podman/blob/main/hack/bats ?

**Comment by ygalblum** [comment]
> What does this do that isn't handled better by https://github.com/containers/podman/blob/main/hack/bats ?

The `Makefile` nothing. The `README.md` however, let's you know that the option exists :). I'll close this PR and open a new one just for pointing users to the `hack` script in the `README.md`

**Pull request closed without merging**